KMaina/Store-Manager-API
# #161396788 Build Out the Register Endpoint

#### What does this PR do?

Have users resgister into the app
#### Description of Task to be completed?

Have administrators log into the app
Have administrators register into the app
#### How should this be manually tested?

- Clone the repo locally
- Create virtual environmnet and run `pytest`
- In postman use the following body to log in

```
{
    "name":"admin",
    "password":"passadmin"
}
```
In postman use the following JSON body to register a new user
```
{
    "name":"Ken Maina",
    "password":"mysecret",
    "confirm":"mysecret"
}
```
#### Any background context you want to provide?

In order to register a user, first you need to log in as an admin

#### What are the relevant pivotal tracker stories?
[#161396788](https://www.pivotaltracker.com/story/show/161396788)
